docs/Deploying-Pega-on-AKS.md

```diff
@@ -446,6 +446,16 @@ Pega added for this purpose in the [pega-helm-charts](https://github.com/pegasys
 
 Make these changes before you begin deploying Pega Platform using Helm charts.
 
+#### Add Support for providing DB credentials using External Secrets Operator
```

Comment:
Add optional, unless it's required:
#### (Optional) Add Support for providing Pega database credentials using an External Secrets Operator

Reply:
Done.